source/renderer/app/components/wallet/settings/ChangeSpendingPasswordDialog.scss

```diff
@@ -39,10 +39,9 @@
         }
       }
 
-      :global
-        .ChangeSpendingPasswordDialog_tooltip.SimpleTooltip_root.TooltipOverrides_root.InlineEditingDropdown-tooltip_root.SimpleTooltip_isShowingOnHover {
+      .tooltip {
         height: 12px;
-        left: 136px;
+        left: 142px;
         position: absolute;
         top: 54px;
         width: 12px;
```

Comment:
Same here, position of the tooltip should be done by the PopOver (and adjusted via Tippy props if necessary) - of course if there is a special case im not yet familiar with, please show me :)

Reply:
Fixed